app/assets/javascripts/controllers/container_topology/container_topology_controller.js

```diff
@@ -51,6 +58,92 @@ angular.module('topologyApp', ['kubernetesUI', 'ui.bootstrap'])
     $interval.cancel(promise);
   });
 
+
+  var refreshProvider = function(d) {
+    $scope.refreshProvider.value = true;
+    document.getElementsByClassName('alert alert-success alert-dismissable refresh_provider')[0].style.display ='block';
+    var url ='/container_topology/refresh_provider/'+ d.item.miq_id;
+    $http.post(url).success(function(data) {
+
```

Comment:
You don't need the `.success` bit if you don't need to do anything there.

But maybe there should be something there?